target/linux/ramips/dts/mt7621_hiwifi_hc5962.dts

```diff
@@ -23,14 +23,15 @@
 	leds {
 		compatible = "gpio-leds";
 
-		led_status: status {
+		led_white: status {
 			label = "hc5962:white:status";
 			gpios = <&gpio0 6 GPIO_ACTIVE_LOW>;
 		};
 
-		system {
+		led_red: system {
```

Comment:
Since the LEDs have "system" and "status" for names, I'd reflect that in the labels:
led_status and led_system
OR
led_status_white and led_system_red